docs/userGuide/syntax/includes.md

```diff
@@ -155,6 +155,61 @@ You can also specify include variables within the `<include>` tag itself by addi
 </div>
 
 If the same variable is defined in a chain of `<include>`s (e.g. `a.md` includes `b.md` includes `c.md`...), variables defined in the top-most `<include>` will take precedence. Global variables (`_markbind/variables.md`) will take precedence over any `<include>` variables.
+This is to allow the outer context to adapt the reused content without changing its actual content.
+
+<panel header="If you are using recursive includes...">
```

Comment:
collapse by default?

Reply:
Do you mean something like this @damithc?
Panels are collapsed by default unless `expanded` is set to true
![image](https://user-images.githubusercontent.com/77204346/223671578-5893a6f6-d52d-4963-b91f-106f8c6c80d2.png)

Reply:
> Panels are collapsed by default unless `expanded` is set to true

@yucheng11122017  Ah, I forgot that. Ignore my comment in that case.
Feel free to use a better title for the panel BTW. The one I suggested was just from the top of my head ...

Reply:
@yucheng11122017 how about changing the title to "Preventing cyclical errors when using multiple includes with same variables"?